Ports/.strip_env.sh

```diff
@@ -1,13 +1,27 @@
 #!/usr/bin/env bash
 
-exec env -i SERENITY_STRIPPED_ENV=1 \
-    HOME="${HOME}" \
-    USER="${USER}" \
-    TERM="${TERM}" \
-    PATH="${PATH}" \
-    EDITOR="${EDITOR:-}" \
-    MAKEJOBS="${MAKEJOBS:-}" \
-    IN_SERENITY_PORT_DEV="${IN_SERENITY_PORT_DEV:-}" \
-    SERENITY_ARCH="${SERENITY_ARCH:-}" \
-    SERENITY_TOOLCHAIN="${SERENITY_TOOLCHAIN:-}" \
-    "${@}"
+environment_variables=(
+    "SERENITY_STRIPPED_ENV=1"
+    "HOME=${HOME}"
+    "USER=${USER}"
+    "TERM=${TERM}"
+    "PATH=${PATH}"
+)
```

Comment:
While we are touching this anyways...

```suggestion
environment_variables=(
    "HOME=${HOME}"
    "PATH=${PATH}"
    "SERENITY_STRIPPED_ENV=1"
    "TERM=${TERM}"
    "USER=${USER}"
)
```